Week 1/id_350/LeetCode_1_350.java

```diff
@@ -0,0 +1,15 @@
+class Solution {
+    public int[] twoSum(int[] nums, int target) {
+        int[] answer = new int[2];
+        for(int i = 0 ;  i < nums.length - 1 ; i++ ){
```

Comment:
Weird to have an extra space before `;`

Reply:
Thanks for comment, i have found it and try to change it.